### PR TITLE
chore(driver): Macro use tracing

### PR DESCRIPTION
**Pull request by refcell**
### Description

Adopts a nice pattern from reth to use tracing macros, removing the need to specify the dep each time.

### CODE REVIEW

**Comment by refcell** [comment]
## 📚 $\text{Stack Overview}$

Pulls submitted in this stack:
* #822 👈
* `main`

_This comment was automatically generated by [`st`](https://github.com/clabby/st)._

File: crates/driver/src/cursor.rs [hunk 1]
```diff
@@ -1,11 +1,12 @@
 //! Contains the cursor for the derivation pipeline.
 
-use crate::TipCursor;
 use alloc::collections::{btree_map::BTreeMap, vec_deque::VecDeque};
 use alloy_consensus::{Header, Sealed};
 use alloy_primitives::{map::HashMap, B256};
 use op_alloy_protocol::{BlockInfo, L2BlockInfo};
```
Comment (clabby) [inline crates/driver/src/cursor.rs:7]:
```suggestion
```

File: crates/driver/src/executor.rs [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
-use alloc::string::ToString;
-use alloy_consensus::{Header, Sealed};
-use alloy_primitives::B256;
-use async_trait::async_trait;
 use core::{
     error::Error,
     fmt::{Debug, Display},
 };
+
```
Comment (clabby) [inline crates/driver/src/executor.rs:7]:
```suggestion
```

File: crates/driver/src/core.rs [hunk 3]
```diff
@@ -5,16 +5,17 @@ use alloy_consensus::{BlockBody, Sealable};
 use alloy_primitives::B256;
 use alloy_rlp::Decodable;
 use core::fmt::Debug;
+
```
Comment (clabby) [inline crates/driver/src/core.rs:8]:
```suggestion
```

**Review by clabby — APPROVED** [review]

**Pull request merged**